CI_YAML.md

```diff
@@ -48,10 +50,13 @@ targets:
 #
 # Tags example:
 # This test will be categorized as host only framework test.
+# Postsubmit runs will be passed "upload: all".
```

Comment:
Not sure what `upload: all` means?

Reply:
I switched it to upload_metrics: true, which is an existing property